### ISSUE TITLE
android 12 (31)

### ISSUE THREAD

**Issue by j0sem4rio** [graphhopper/graphhopper-navigation-example-legacy#52]
I'm trying to update my application to Android SDK 31 but I'm having an issue with graphhopper
Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.

I already tried to solve this problem with Override, but I couldn't.

**Comment by boldtrn** [comment]
Yes, this is a known problem. Feel free if you want to open a PR for a fix. Otherwise you might be interested in having a look at https://github.com/maplibre/maplibre-navigation-android - we have fixed this bug a few months ago there.
